### ISSUE TITLE
Calling rpc crashes editor

### ISSUE THREAD

**Issue by vojtechmusilek** [FirstGearGames/FishNet#230]
Unity version: 2021.3.4f1 or 2021.3.16f1 (tested on both)
Fish-Networking version: 3.1.3 or 3.1.4 (tested on both)

**Describe the bug**
When ObserversRpc in Example.cs is called from Runner.cs the editor crashes.
But when List<int> is commened/removed from Data.cs it works for some reason.

It was last working on 3.0.0 (and maybe on 2.6.6, but not sure about that one)

**To Reproduce**
1. Download zipped example project: [Fishnet314EditorCrashBug.zip](https://github.com/FirstGearGames/FishNet/files/10459562/Fishnet314EditorCrashBug.zip)
2. Open in editor 2021.3.4f1 or 2021.3.16f1 (not sure if this is editor specific, maybe it will crash on any version)
3. Open scene: Assets/Scenes/SampleScene
4. Run and it will crash editor (It crashes when it runs line 22 in Runner.cs)
6. (You can also comment/remove List<int> in Data.cs and then it will not crash)

**Expected behavior**
Not crash editor.

**Comment by FirstGearGames** [comment]
Thank you, this will be tested immediately.

**Comment by FirstGearGames** [comment]
Confirmed. I will have this fixed soon but if you need an immediate fix change your collection from a property to a field.
eg:
```
public List<int> TheList {get; set;}
```
to this..
```
public List<int> TheList;
```

**Comment by FirstGearGames** [comment]
Fixed in 3.1.5

**Comment by vojtechmusilek** [comment]
I can confirm its now fixed. Thanks for the very quick fix.
